source/FallingBlock.hx

```diff
@@ -0,0 +1,67 @@
+package;
+
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import haxe.Timer;
+
+/**
+ * A block that falls after being stepped on
+ * @author TR Nale
+ */
+class FallingBlock extends Block
```

Comment:
That might be better handled by moving platforms.

Either way, I think that is best left as a future feature.